app/scripts/modules/core/src/help/help.contents.ts

```diff
@@ -118,7 +125,16 @@ module(HELP_CONTENTS, [])
         <p>The constraint values may be supplied as regex.</p>
     `,
     'pipeline.config.trigger.pubsub.payloadConstraints': `
-        <p>When provided, only a pubsub message with a payload containing at least the specified key/value pairs will be allowed to trigger this pipeline. For example, if you wanted to lockdown the systems/users that can trigger this pipeline via this pubsub subscription, you could require the key "secret" and value "something-secret" as a constraint.</p>
+        <p>
+          When provided, only a pubsub message with a payload containing at least the specified
+          key/value pairs will be allowed to trigger this pipeline. For example, if you wanted
+          to lockdown the systems/users that can trigger this pipeline via this pubsub
+          subscription, you could require the key "secret" and value "something-secret" as a constraint.
+        </p>
+        <p>
+          The key/value pairs are matched against the unprocessed payload body, prior to any
+          transformation using the jinja template in the pubsub configuration.
```

Comment:
I think `in the pubsub subscription configuration` would be a little clearer, since the templates are configured per subscription.